src/test/java/com/thealgorithms/datastructures/hashmap/hashing/MajorityElementTest.java

```diff
@@ -0,0 +1,48 @@
+package com.thealgorithms.datastructures.hashmap.hashing;
+
+import com.thealgorithms.datastructures.hashmap.hashing.MajorityElement;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import java.util.ArrayList;
+
+public class MajorityElementTest{
+    @Test
+    void testMajorityWithSingleMajorityElement() {
```

Comment:
There's no majority element in this test

Reply:
added